backend/src/federation/index.ts

```diff
@@ -36,19 +32,61 @@ export const startDHT = async (
     logger.info(`keyPairDHT: publicKey=${keyPair.publicKey.toString('hex')}`)
     logger.debug(`keyPairDHT: secretKey=${keyPair.secretKey.toString('hex')}`)
 
+    const apiList = {
+      apiVersions: Object.values(ApiVersionType).map(function (apiEnum) {
+        return { api: apiEnum, url: CONFIG.FEDERATION_COMMUNITY_URL }
+      }),
```

Comment:
why do you need the subfield apiVersions. I still do not understand this?
Why dont we use `apiList = []` the apiList to be an array?

```
const apiList = [
{ api: 'v1_0', url: 'https://localhost:4000/api/v1_0/' },
{ api: 'v1_1', url: 'https://localhost:4000/api/v1_1/' },
{ api: 'v2_0', url: 'https://localhost:4000/api/v2_0/' },
]
```

If I read [this](https://www.w3schools.com/JS/js_json_arrays.asp) correctly `'["Ford", "BMW", "Fiat"]'` is a valid json string,

Reply:
As in a previous version i created the types

type CommunityApi = {
  api: string
  url: string
}
type CommunityApiList = {
  apiVersions: CommunityApi[]
}
to have structured objects. To benefit of the CommunityApi-Type with only one attribute of type CommunityApi[] base on an easier check of the parsed JSON-string after received data. Without the CommunityApiList-Type you have to check on an array of ComunityApi-Type entries with several attributes.

I reintroduced the types back, because on my opinion working with more type-safetiness in the code increases the quality more than focus on simply reducing the amount of lines of code.